.gitignore

```diff
@@ -20,3 +20,4 @@ setuptools.egg-info
 .idea/
 .pytest_cache/
 .mypy_cache/
+/setuptools/tests/config/downloads
```

Comment:
Instead of this, I suggest creating `setuptools/tests/config/downloads/.gitignore` that contains:
```gitignore
*
!.gitignore
```